exchanges/gateio/gateio_wrapper.go

```diff
@@ -148,6 +151,12 @@ func (g *Gateio) SetDefaults() {
 	g.WebsocketResponseMaxLimit = exchange.DefaultWebsocketResponseMaxLimit
 	g.WebsocketResponseCheckTimeout = exchange.DefaultWebsocketResponseCheckTimeout
 	g.WebsocketOrderbookBufferLimit = exchange.DefaultWebsocketOrderbookBufferLimit
+	g.WsChannelsMultiplexer = &WsMultiplexer{
```

Comment:
Since the method, SendMessageReturnResponse is not returning the message appropriately. I think we should keep this implementation.

It has no CPU overhead since every **"ID"** sent with the request comes with the response; no looping is needed. Just select the channel using the ID and pass the message.

I can change the Implementation If you don't find It relevant.